src/lib/errors.js

```diff
@@ -0,0 +1,13 @@
+/* @flow */
+
+import { ERROR_CODE } from '@paypal/sdk-constants';
```

Comment:
I noticed the other sdk-constants import directly from the src folder:
```suggestion
import { ERROR_CODE } from '@paypal/sdk-constants/src';
```